hphp/runtime/vm/jit/smashable-instr-arm.cpp

```diff
@@ -337,22 +337,31 @@ ConditionCode smashableJccCond(TCA inst) {
  *       sequence is a full jcc even though a jcc actually uses the same
  *       sequence as a jmp in its implementation.
  */
-TCA getSmashableFromTargetAddr(TCA addr) {
+hphp_hash_set<vixl::Instruction*> getSmashablesFromTargetAddr(TCA addr) {
```

Comment:
I think these days we just prefer `std::unordered_set` here since newer compilers don't force-malloc when they're constructed.

That said, maybe a `std::array<TCA,3>` would be better here so that we return on the stack instead of allocating memory at all?

Reply:
Will do.